pkg/reconciler/v1alpha1/taskrun/resources/pod_test.go

```diff
@@ -435,3 +435,140 @@ func TestMakeWorkingDirScript(t *testing.T) {
 		})
 	}
 }
+
+func TestInitOutputResourcesDefaultDir(t *testing.T) {
+	names.TestingSeed()
+
+	randReader = strings.NewReader(strings.Repeat("a", 10000))
+	defer func() { randReader = rand.Reader }()
+
+	for _, c := range []struct {
+		desc         string
+		trs          v1alpha1.TaskRunSpec
+		ts           v1alpha1.TaskSpec
+		bAnnotations map[string]string
+		want         *corev1.PodSpec
+		wantErr      error
+	}{{
+		desc: "task-with-output-image-resource",
```

Comment:
hmm if we just have one test case, maybe we don't need the for loop? (might make the test a bit simpler to read)

Reply:
will actually move it inside the already existing table test, no need to duplicate this separately